src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -600,6 +604,8 @@ public SscsCaseData(@JsonProperty(value = "ccdCaseId", access = JsonProperty.Acc
         this.panel = panel;
         this.evidenceReceived = evidenceReceived;
         this.urgentCase = urgentCase;
+        this.urgentHearingRegistered = urgentHearingRegistered;
```

Comment:
What does this field represent? If a date can we name it `urgentHearingRegisteredDate` or something less ambiguous?

Reply:
Its represent the date value, as mentioned in the ticket the name should be urgentHearingRegistered